AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -67,9 +67,12 @@ class MeetingModel: NSObject {
                     videoModel.isLocalVideoActive = true
                 }
                 if activeMode == .video {
-                    videoModel.addAllRemoteVideosInCurrentPageExceptUserPausedVideos()
+                    videoModel.updateAllRemoteVideosInCurrentPageExceptUserPausedVideos()
+                } else if activeMode == .screenShare{
+                    videoModel.addContentShareVideoSource()
                 }
             }
+            videoModel.updateVideoSourceSubscription()
```

Comment:
should this be only called when activeMode is either `.video` or `.screen`?

Reply:
When switching to other mode, the function `videoModel.unsubscribeAllRemoteVideos()` would be called and it also requires `updateVideoSourceSubscription()` to take effect. Also, if there is nothing to update for video source subscription, the function `updateVideoSourceSubscription()` would just return and it would not affect the efficiency.